device/src/region/eu868/mod.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(dead_code)]
+use super::*;
+use lorawan_encoding::maccommands::ChannelMask;
+
+const JOIN_CHANNELS: [u32; 3] = [868_100_000, 868_300_000, 868_500_000];
+
+#[derive(Default)]
+pub struct EU868 {
+    subband: Option<u8>,
+    last_tx: (u8, u8),
+    cf_list: Option<[u32; 5]>,
+}
+
+impl EU868 {
+    pub fn new() -> EU868 {
+        Self::default()
+    }
+}
+
+use super::JoinAccept;
+
+impl RegionHandler for EU868 {
+    fn process_join_accept<T: core::convert::AsRef<[u8]>, C>(
+        &mut self,
+        join_accept: &super::DecryptedJoinAcceptPayload<T, C>,
+    ) -> JoinAccept {
+        let mut new_cf_list = [0, 0, 0, 0, 0];
+        if let Some(cf_list) = join_accept.c_f_list() {
+            for (index, freq) in cf_list.iter().enumerate() {
+                new_cf_list[index] = freq.value();
+            }
+        }
+        self.cf_list = Some(new_cf_list);
```

Comment:
What is the advantage of having cf_list in the region parameters? My first thought would have been that the device copies this in its state while starting and never looks back to the region. Then such changes would happen on the device state.

Reply:
As I mentioned before, I consider this to be part of "regional logic" as opposed to "core logic". 

Basically, anything where the idea comes from [the regional spec](https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf) will go in regions and anything that's part of the [core spec](https://lora-alliance.org/resource_hub/lorawan-specification-v1-0-3/) goes into the device session/nosession state machines.